Tests/Runtime/Annotations/InputFieldAnnotationTest.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2023 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using TestHelper.Attributes;
+using TestHelper.Monkey.Annotations.Enums;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace TestHelper.Monkey.Annotations
+{
+    [TestFixture]
+    [GameViewResolution(GameViewResolution.VGA)]
+    public class InputFieldAnnotationTest
+    {
+        private const string TestScene = "Packages/com.nowsprinting.test-helper.monkey/Tests/Scenes/InputFields.unity";
+        private GameObject _annotationAttachedGameObject;
+
+        private class InputFieldValidator : MonoBehaviour
+        {
+            private InputField _inputField;
+            private Regex _pattern;
+
+            public void SetValidPattern(string pattern)
+            {
+                _pattern = new Regex(pattern);
+            }
+
+            private void Start()
+            {
+                _inputField = GetComponent<InputField>();
+            }
+
+            private void Update()
+            {
+                if (_inputField.text == string.Empty)
+                {
+                    return;
+                }
+
+                if (!_pattern.IsMatch(_inputField.text))
+                {
+                    Debug.LogError($"{gameObject.name}: {_inputField.text} is not match pattern.");
+                }
+            }
+        }
```

Comment:
I think we should locate it on the directory "TestDoubles."

Reply:
Fixed in 74deb51

Reply:
I think after commit.
This class is not test double.
Move to another namespace, e.g., validators?

Reply:
Since there is no other suitable namespace, I will put it on TestDoubles.